packages/node-xmpp/test/tls-test.js

```diff
@@ -118,7 +118,9 @@ describe('TLS', function () {
         done(new Error('should not allow any authentication'))
       })
 
-      cl.once('error', function () {
+      cl.once('error', function (err) {
+        // assert(err instanceof Error) FIXME should be an instance of Error
+        assert.equal(err.toString(), 'XMPP authentication failure')
         done()
       })
     })
```

Comment:
TODO later those tests should be moved to a private sub package "node-xmpp-tests"